### ISSUE TITLE
Possible problems with core-js ?

### ISSUE THREAD

**Issue by tomitrescak** [thereactivestack/kickstart-flowrouter#9]
Hi, I'm trying to use the flowrouter package, but it is not allowing to write more sophisticated queries which fail during "check". Here's what to do to make it fail:

1. Download a fresh repo
2. Modify "todo-subscriptions" by adding `{limit: 2}` to the query
3. When reloaded following error appears:

```
I20151026-14:34:06.381(11)? Exception from sub tasks id qShhrhNmmSwFihYks Error: Match error: Failed Match.OneOf or Match.Optional validation
I20151026-14:34:06.381(11)?     at checkSubtree (packages/check/packages/check.js:252:1)
I20151026-14:34:06.382(11)?     at check (packages/check/packages/check.js:40:1)
I20151026-14:34:06.382(11)?     at [object Object]._.extend._getFindOptions (packages/mongo/collection.js:244:1)
I20151026-14:34:06.382(11)?     at [object Object]._.extend.find (packages/mongo/collection.js:280:1)
I20151026-14:34:06.382(11)?     at [object Object].eval [as _handler] (eval at <anonymous> (server.js:1206:1), <anonymous>:12:47)
I20151026-14:34:06.382(11)?     at maybeAuditArgumentChecks (livedata_server.js:1692:12)
I20151026-14:34:06.382(11)?     at [object Object]._.extend._runHandler (livedata_server.js:1023:17)
I20151026-14:34:06.382(11)?     at [object Object]._.extend._startSubscription (livedata_server.js:842:9)
I20151026-14:34:06.382(11)?     at [object Object]._.extend.protocol_handlers.sub (livedata_server.js:614:12)
I20151026-14:34:06.383(11)?     at livedata_server.js:548:43
I20151026-14:34:06.383(11)? Sanitized and reported to the client as: Match failed [400]
```

I have debugged this on server and there is a problem with number constructor .. could this be a core-js problem?

Here is my todo-subscriptions

```
import Tasks from 'TodoApp/collections/Tasks';

// This code only runs on the server
Meteor.publish('tasks', function () {
  console.log("Limited")
  return Tasks.find({
    $or: [
      { private: {$ne: true} },
      { owner: this.userId }
    ]
  }, { limit: 2 });
});
```

**Comment by tomitrescak** [comment]
Weird part is that on kickstart-simple performing the very same steps does not lead to error :/

**Comment by eXon** [comment]
reat-router-ssr is actually patching this bug. I have a good idea of what the problem is so I'll try to fix it for good.

**Comment by eXon** [comment]
This has been fixed in webpack:webpack 0.3.1

It uses meteor polyfill of ecmascript instead of babel polyfill.
